docker/run_dependency_checks.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Runs dependency checks. Must be run from within the crontabber docker container.
+
+set -e
+
+./scripts/socorro crontabber --reset-job=socorro.cron.jobs.monitoring.DependencySecurityCheckCronApp
+./scripts/socorro crontabber --job=socorro.cron.jobs.monitoring.DependencySecurityCheckCronApp
```

Comment:
These commands output a ton of logging with environment variable values, and I couldn't figure out a way to suppress them. It'd be really nice if we could make the output less noisy.

Reply:
OMG. SOOOO MUCH LOGGING!

I wrote up a bug to redo how logging works and either we can tackle this directly in that bug or at least pave the way for a better world.